code/modules/projectiles/guns/ballistic/automatic.dm

```diff
@@ -134,7 +134,7 @@
 
 /obj/item/gun/ballistic/automatic/m90
 	name = "\improper M-90gl Carbine"
-	desc = "A three-round burst 5.56 toploading carbine, designated 'M-90gl'. Has an attached underbarrel grenade launcher which can be toggled on and off."
+	desc = "A three-round burst 5.56x45mm toploading rifle, designated 'M-90gl'. Has an attached 40mm underbarrel grenade launcher which can be toggled on and off."
```

Comment:
why x45? That's a huge round that wouldn't fit in a top-mounted magazine. The real p90 uses 5.7x28

Reply:
it's flavourful but he got it wrong because he's not a gun autist

Reply:
The M90-gl is not a P90.

Reply:
Have you seen the sprite?

Reply:
Yeah the sprite looks like a P90 with a grenade launcher, so? It's not one.

Reply:
> The M90-gl is not a P90.

the point is you added the x45mm measurement for no reason and it makes no sense given the dimensions of what a normal gun looks like